modules/library/metadata/src/test/java/org/geotools/util/GeoToolsUnitFormatRoundTripTest.java

```diff
@@ -45,21 +45,22 @@
     public static Collection<Object[]> data() {
         return Arrays.asList(
                 new Object[][] {
+                    {GRAM},
                     {GRAM.divide(1_000_000)},
                     {GRAM.divide(1_000_000).divide(METRE.pow(3))},
                     {GRAM.divide(1_000_000_000).divide(METRE.pow(3))},
                     {GRAM.divide(1_000_000).divide(METRE.pow(3))},
                     {METRE.pow(2)},
                     {METRE.pow(3)},
-                    {SI.MOLE.divide(1_000_000)},
+                    {MOLE.divide(1_000_000)},
```

Comment:
No division between SI and NonSI anymore eh?

Reply:
No they both still exist in si.uom:si-units:2.0.1 but it seemed like some things that were considered non-SI before were now considered SI.  Possibly the document they base the types on was updated since the original Units of Measure 1.0 implementation.